app/controllers/users/passwords_controller.rb

```diff
@@ -25,6 +26,15 @@ def update
 
     private
 
+    def pii_requested_but_locked?
+      if UserDecorator.new(current_user).identity_verified? &&
+         user_session[:decrypted_pii].blank?
+
```

Comment:
We tend to do `current_user.decorate` instead, that might shorten this to be able to fit on one line?

```suggestion
      if current_user.decorate.identity_verified? && user_session[:decrypted_pii].blank?
```